src/util/hasValue.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * ensures that the value passed in is not null or undefined or is not an empty
```

Comment:
1. i'm not sure `ensures` is the correct wording it does not ensure anything. it only 'checks'
2. again we are taking about the 'value passed' this is the same as 'accepts'
3. use: 'Checks if a value is not `null`, `undefined` or an empty string'